static/js/place/chart.tsx

```diff
@@ -194,33 +162,6 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
   }
 
   componentDidUpdate(): void {
-    if (!this.state.display) {
-      return;
-    }
-    const dp = this.state.dataPoints;
-    const dg = this.state.dataGroups;
-    if (
-      (dp && dp.length === 0) ||
-      (dg &&
-        (dg.length === 0 || (dg.length === 1 && dg[0].value.length === 0))) ||
-      (this.props.config.chartType == "LINE" &&
-        dg &&
-        dg.reduce((accum, group) => {
```

Comment:
this was a check to prevent showing line charts with single points #200 (and another place that I didn't record had multiple statvars with a single point)